docs/source/examples/checks/methodology/single_feature_contribution_train_test.ipynb

```diff
@@ -5,123 +5,83 @@
    "id": "21c5d5bc",
```

Comment:
<p>First sentence is confusing imo. I suggest</p><p>The check calculates the PPS for the top features of each dataset, and elaborates how to interpret the results.</p><p> </p><p>In addition, I don't like when we write "we" (second sentence).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>